src/core/style/_variables.scss

```diff
@@ -31,3 +31,24 @@ $swift-ease-in: all $swift-ease-in-duration $swift-ease-in-timing-function !defa
 $swift-ease-in-out-duration: 0.5s !default;
 $swift-ease-in-out-timing-function: $ease-in-out-curve-function !default;
 $swift-ease-in-out: all $swift-ease-in-out-duration $swift-ease-in-out-timing-function !default;
+
+// Progress Bar Timing functions:
+// $progress-bar-primary-indeterminate-translate-step-1 has no timing function.
+$progress-bar-primary-indeterminate-translate-step-2: cubic-bezier(.5, 0, .701732, .495819) !default;
```

Comment:
Seeing these, I think they're actually specific enough to go in the progressbar scss.
(I think we talked about this in person, I didn't know how specific they were).